src/domain/synthetics/positions/utils.ts

```diff
@@ -245,3 +246,47 @@ export function formatLeverage(leverage?: BigNumber) {
 
   return `${formatAmount(leverage, 4, 2)}x`;
 }
+
+export function getEstimatedLiquidationTime(
+  position: PositionInfo,
+  minCollateralUsd: BigNumber | undefined,
+  period = "1h"
```

Comment:
If this function only counts hours, I don't think we need to add the "period" parameter

Reply:
maybe rename function to getEstimatedLiquidationTimeInHours?

Reply:
Done!